### ISSUE TITLE
Right-side toolbar disappears on zoom > 100%

### ISSUE THREAD

**Issue by gtfortytwo** [edge-reflow/issues#100]
The right-side toolbar (with Edge Inspect, Asset Lib, Pages, PS) disappears when the canvas zoom is set to greater than 100%.  And it appears that there is no menu-bar or keyboard equivalent for these commands, so the only way to access them appears to be to drop the zoom level back down.

Using version 0.31.13658 (preview) on Win 8 x64.

Here's a normal screencap at 100%:

![image](https://f.cloud.github.com/assets/718269/1227592/11d1a594-2795-11e3-85fd-a244f4f92c1c.png)

... and an example of the breakage when zoomed to 200%:

![image](https://f.cloud.github.com/assets/718269/1227617/63a5ecf4-2795-11e3-9ed3-c08213e2c1a4.png)

The problem occurs whether or not the canvas stretches the full width of app window.  It also still happens if the canvas section is scrolled all the way to the right.

**Comment by jtlafferty** [comment]
yep, we are aware of this bug :(  We will report back when a resolution is in progress. Thanks!

**Comment by gtfortytwo** [comment]
I did look through the github issues, sorry if it was a dupe.

**Comment by jtlafferty** [comment]
No, no! It was not a duplicate. We have our own bug list and a backlog where we track things our own team has seen.  Sometimes, we don't always know how many customers are actually hitting those issues. We appreciate you logging the bug which tells us this is a real problem customers are running into. We are discussing solutions now.

**Comment by ElPepe101** [comment]
Back again in 2014, same issue; Version 0.47.17127 (preview) w/ Mac OSX Mavericks.
